### ISSUE TITLE
The task runs but doesn't change the icon

### ISSUE THREAD

**Issue by sturlath** [damienaicheh/azure-devops-launch-icon-badge#2]
Hi the task runs and seems to edit the icon but it doesn´t change it either in AppCenter or when I install the aab file.

This is the output from AzureDevOps

> Starting: LaunchIconBadge
==============================================================================
Task         : Launch Icon Badge
Description  : Draw badge in your application launch icon.
Version      : 1.0.3
Author       : Damien Aicheh
Help         : 
==============================================================================
Edition succeeded for: d:\a\1\s\ST.MathTester.Multiplication.Android\obj\Release\90\res\drawable\icon.png
Edition succeeded for:d:\a\1\s\ST.MathTester.Multiplication.Android\Resources\drawable\Icon.png
Finishing: LaunchIconBadge

And this is the task

     - task: LaunchIconBadge@1
       inputs:
          sourceFolder: '$(Build.SourcesDirectory)\ST.MathTester.Multiplication.Android' # Optional. Default is: $(Build.SourcesDirectory)
          contents: '**/*.png' # Optional. Default is:  '**/icon.png'
          bannerVersionNamePosition: 'bottomRight' # Options: topLeft, topRight, bottomRight, bottomLeft. Default is: 'bottomRight'
          bannerVersionNameText: 'Prerelease'  # Optional. Default is: ''
          bannerVersionNameColor: '#C5000D' # Optional. Default is: '#C5000D'
          bannerVersionNameTextColor: '#FFFFFF' # Optional. Default is: '#FFFFFF'
          bannerVersionNumberPosition: 'top' # Optional. top, bottom, none. Default is: 'none'
          bannerVersionNumberText: '$(version_prefix).$(Build.BuildId)' # Optional. Default is: ''
          bannerVersionNumberColor: '#34424F' # Optional. Default is: '#34424F'
          bannerVersionNumberTextColor: '#FFFFFF' # Optional. Default is: '#FFFFFF' 

I'm using Nicks Randolphs https://nicksnettravels.builttoroam.com/pipeline-template-launchiconbadge/ setup but I can't see that its causing any problems.

Is this in anywhy related to the png them selfs? This was just a temp icon I took from the internet until I designed one.

![Icon](https://user-images.githubusercontent.com/2386572/79360218-2213bd00-7f33-11ea-8b60-4120d384e76a.png)
![icon](https://user-images.githubusercontent.com/2386572/79360266-2fc94280-7f33-11ea-9edc-53f3e932ab8b.png)

**Comment by damienaicheh** [comment]
Hi! When I look at your logs there is something strange to me, in the first line you are editing image inside the obj folder with an image called with a (little 'i') and in the second log you are editing an image inside the Resouces directories (with a capital 'i') which is the good way. You need to edit the icons inside your Resources folder not after in the obj or bin folder because they are probably already packaged in your aab or apk.

I don't know which vmImage you use but for the sourceFolder you use \ and after you use / make sur to use it depending on the OS you use.

So what I suggest to you:
- Make sure you edit the icons only in the resources folder before the build start
- Fix the / and \
- If you are not sure about your image create a little repos with github and upload a set of icons and run my extension and you will see if there is a problem with your image.

Bonus:
-  Make sure you add an icon to each of your folders (better for aab because it split the resources to generate the correct apk depending on each devices needs)

Hope you will find your answer!

**Comment by sturlath** [comment]
Ok thank you.. had to comb over what I was doing few times... 

Firstly I put the task under "prePublish:" and not "preBuild"

And then the icon I was using for my app was NOT Icon/icon.png but another one I had added in different xxpi sizes. I had done this long ago and just assumed that it was the icon one (its the same image).

I fixed / and \ and now everything works. 

Thank you for your help and sorry for me not spotting these obvious things :-) Have a great day and love the outcome of my new app icon!
